src/pages/projectSettings/tabs/NotificationsTab/getFormSchema.tsx

```diff
@@ -93,3 +88,22 @@ export const getFormSchema = (
     },
   };
 };
+
+const HelpText: React.VFC = () => {
+  const spruceConfig = useSpruceConfig();
+  const slackName = spruceConfig?.slack?.name;
+  console.log(`spruceConfig: ${spruceConfig}`);
+  console.log(spruceConfig);
+
+  return (
+    <Description>
+      Private slack channels may require further Slack configuration.
+      {slackName && (
+        <div>
+          Invite evergreen to your private Slack channels by running{" "}
```

Comment:
Not sure if this is correct but it may be more clear
```suggestion
          Invite evergreen-bot to your private Slack channels by running{" "}
```

Reply:
I left if more vague because at the present moment it's a bot, but very soon it'll be the slack app.